src/State/BrugerFunktionerProvider.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+namespace App\State;
+
+use ApiPlatform\Metadata\Operation;
+use ApiPlatform\State\ProviderInterface;
+use App\Exception\ProviderException;
+use App\Repository\FunktionsDataRepository;
+
+readonly class BrugerFunktionerProvider implements ProviderInterface
+{
+    public function __construct(private FunktionsDataRepository $funktionsDataRepository)
+    {
+    }
+
+    public function provide(Operation $operation, array $uriVariables = [], array $context = []): object|array|null
+    {
+        if (!isset($uriVariables['id'])) {
+            throw new ProviderException('Could not find id in uri');
```

Comment:
Is this the right exception to throw? Shouldn't it be something like “invalid request” or whatever API Platform throws if no ID is provided?

Reply:
Added custom `InvalidProviderRequestException`.